main.go

```diff
@@ -73,7 +82,23 @@ func main() {
 		os.Exit(1)
 	}
 
-	registerRatioController(mgr, *memoryCPURatio, *organizationLabel)
+	kubeInformer := kubeinformers.NewSharedInformerFactory(kubernetes.NewForConfigOrDie(mgr.GetConfig()), 15*time.Minute)
```

Comment:
Why do we use informers instead of just using the client of the manager, which should handle all this for us?

Reply:
I might have been lazy. I'll see how much code it is to change the interfaces.

Reply:
I've removed the informer for good now 👍